shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx

```diff
@@ -298,7 +290,9 @@ const AddModal = (props: IProps) => {
     switch (bodyType) {
       case noneType:
         updater.textOrJson('');
-        updater.headers({});
+        updater.headers({
+          ...headers,
```

Comment:
What's the point of this

Reply:
When there is a value in the headers and the type of the body is switched to none, the value of the original headers remains unchanged